tests/snmp/test_snmp_psu.py

```diff
@@ -7,8 +7,8 @@
 ]
 
 @pytest.mark.bsl
-def test_snmp_numpsu(duthosts, rand_one_dut_hostname, localhost, creds):
-    duthost = duthosts[rand_one_dut_hostname]
+def test_snmp_numpsu(duthosts, enum_supervisor_dut_hostname, localhost, creds):
+    duthost = duthosts[enum_supervisor_dut_hostname]
```

Comment:
I think this change will make the test case only applicable to supervisor node. It can't be executed for pizza box DUT.